lib/plugins/aws/invokeLocal/index.js

```diff
@@ -130,8 +130,9 @@ class AwsInvokeLocal {
     }
 
     if (runtime === 'python2.7' || runtime === 'python3.6') {
-      const handlerPath = handler.split('.')[0];
-      const handlerName = handler.split('.')[1];
+      const handlerComponents = handler.split(/[.]+/);
```

Comment:
Splitting on consecutive dots might lead to unwanted results here, e.g.
```
my..my.myTest => handlerPath=my.my    handlerName=myTest
```
So the original double dot in the path is swallowed.

Splitting on single dots only will fix that:
`const handlerComponents = handler.split(/\./);`

Reply:
It is swallowed, but I don't think that's a valid module path.

For example, using __import__ directly:
```python
In [6]: from my..my import myTest
  File "<ipython-input-6-262523a38747>", line 1
    from my..my import myTest
            ^
SyntaxError: invalid syntax
```